.travis/collection.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -e
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+. "${SCRIPTDIR}/config.sh"
+
+# Collection commands that are run when `tox -e collection`:
+role=$(basename "${TOPDIR}")
+toxworkdir=${1:-${TOPDIR}/.tox}
+cd "${toxworkdir}"
+toxworkdir=$(pwd)
+envlist=${2:- "yamllint,py38,shellcheck,custom"}
+
+# TEMPORARY will be replaced with the next line.
+# git clone https://github.com/linux-system-roles/auto-maintenance
+# or
+# git clone --branch some_stable_tag https://github.com/linux-system-roles/auto-maintenance
+
+# TEMPORARY
+git clone --single-branch --branch role2collection https://github.com/nhosoi/auto-maintenance
+# TEMPORARY
+
+git clone https://github.com/linux-system-roles/template
+
+COLLECTION_SRC_PATH=${TOPDIR}/.. COLLECTION_DEST_PATH=${toxworkdir} COLLECTION_ROLE=${role} python auto-maintenance/lsr_role2collection.py > "${toxworkdir}"/collection.out 2>&1
+
+COLLECTION_SRC_PATH=${toxworkdir} COLLECTION_DEST_PATH=${toxworkdir} python auto-maintenance/lsr_role2collection.py --tox >> "${toxworkdir}"/collection.out 2>&1
```

Comment:
Is there some reason to use env. vars. here instead of e.g. `--src-path`, `--dest-path`, etc.?

Reply:
No, no reasons. We can use the options.